packages/ubuntu_wsl_setup/windows/runner/.clang-format

```diff
@@ -0,0 +1,8 @@
+# Format Style Options - Created with Clang Power Tools

```

Comment:
[flutter/plugins](https://github.com/flutter/plugins/blob/main/.clang-format) and [flutter/packages](https://github.com/flutter/packages/blob/main/.clang-format) both have:
```
BasedOnStyle: Google
---
Language: Cpp
DerivePointerAlignment: false
PointerAlignment: Left
```
could you also apply the format on all files so later changes won't include unrelated formatting changes?

Reply:
i'd propose that we put the `.clang-format` all the way to the repo root. later on, we could even make the CI format C++ code too i.e. run `flutter-plugin-tools format` instead of just `flutter format`

Reply:
I really like the idea!

Reply:
> [flutter/plugins](https://github.com/flutter/plugins/blob/main/.clang-format) and [flutter/packages](https://github.com/flutter/packages/blob/main/.clang-format) both have:
> 
> ```
> BasedOnStyle: Google
> ---
> Language: Cpp
> DerivePointerAlignment: false
> PointerAlignment: Left
> ```
> 
> could you also apply the format on all files so later changes won't include unrelated formatting changes?

I started with that and in between rebases I forgot to edit the commit message. The app template code doesn't seem to follow any specific standard so adopting one that favors one set of files causes format changes to others. The solution really is choosing one thing and reformatting the whole source set at once. That makes more sense on its own PR.